.gitlab-ci.yml

```diff
@@ -6,11 +6,10 @@ services:
 variables:
   # @TODO: should be codeception/codeception
   ALPINE_IMAGE: herloct/codeception-test:$CI_BUILD_REF_NAME-alpine
```

Comment:
Should we use Secret Variables for `DOCKERHUB_ALPINE_IMAGE`?
So we could merge it later without changing the `.gitlab-ci.yml`.